AppDashboard/static/plugins/forms/tiny_mce/plugins/table/js/table.js

```diff
@@ -242,16 +242,16 @@ function insertTable() {
 	} else
 		inst.execCommand('mceInsertContent', false, html);
 
-	tinymce.each(dom.select('table[data-mce-new]'), function(node) {
+	tinymce.each(dom.select('table[data-mce-new]'), function(node) {

```

Comment:
In this case, you can do `git diff HEAD master AppDashboard/static/plugins/forms/tiny_mce/plugins > plugins.diff` and then `git apply plugins.diff`. You can remove the diff file after. That will undo any changes your branch introduced from master for that directory.

If you notice before pushing the branch publicly, you can undo a commit with `git reset HEAD~1`.